desk/src/components/List/ListFilters.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+	<NestedPopover>
+		<template #target>
+			<Button label="Filter">
+				<template #prefix>
+					<FeatherIcon name="filter" class="h-3.5" />
+				</template>
+				<template v-if="filters?.length" #suffix>
+					<div
+						class="flex h-5 w-5 items-center justify-center rounded bg-gray-900 pt-[1px] text-2xs font-medium text-white"
+					>
+						{{ filters?.length }}
+					</div>
+				</template>
+			</Button>
+		</template>
+
+		<template #body="{ close }">
+			<div class="rounded-lg border border-gray-100 bg-white p-4 shadow-xl">
+				<div
+					v-if="filters?.length"
+					v-for="({ fieldname, fieldtype, operator, value, options }, i) in filters"
+					class="mb-3 flex items-center gap-2"
+				>
+					<div class="w-[9rem]">
+						<Autocomplete
+							:body-classes="'w-[15rem]'"
+							:model-value="fieldname"
+							:options="allFilterableFields"
+							@update:model-value="(val) => updateField(val, i)"
+						/>
+					</div>
+					<div class="w-[7rem]">
+						<FormControl
+							type="select"
+							:model-value="operator"
```

Comment:
Let's also set the first operator from the list as default here. 

![default](https://github.com/frappe/frappe/assets/24353136/3a3d4257-ecf6-4342-af58-9123ee1d8a82)

Also - In, Not In operators should render a multiselect list control if fieldtype is Link

<img width="554" alt="image" src="https://github.com/frappe/frappe/assets/24353136/702d4a74-7ed6-456e-be31-1a01dc9a9719">